.gitignore

```diff
@@ -6,3 +6,4 @@ composer.phar
 .php-cs-fixer.php
 .php-cs-fixer.cache
 .phpunit.result.cache
+.idea
```

Comment:
Please remove this.
It should be added to your global .gitignore

Reply:
Okay, I'll do it later today.